src/domUpdates.js

```diff
@@ -53,6 +55,33 @@ const showSearchResults = (userValue, names, images, ids) => {
   }
 }
 
+const displayAllRecipes = (recipeData) => {
+  hideAllPages();
+  allRecipesView.classList.remove("hidden")
+  const recipeDataAlpha = recipeData.sort((a, b) => {
+    const nameA = a.name.toUpperCase();
+    const nameB = b.name.toUpperCase();
+    if (nameA < nameB) {
+      return -1;
+    }
+    if (nameA > nameB) {
+      return 1;
+    }
+    return 0;
+  });
+  const recipeIds = getItems(recipeDataAlpha, 'id')
+  const recipeNames = getItems(recipeDataAlpha, 'name')
+  const recipeImages = getItems(recipeDataAlpha, 'image')
+  recipeData.forEach((_, i) => {
+    allRecipesSection.innerHTML += `
```

Comment:
we need to reset innerHTML to be empty or the html will populate over and over again